google/generativeai/types/retriever_types.py

```diff
@@ -174,7 +175,7 @@ class Corpus:
 
     def create_document(
         self,
-        name: Optional[str] = None,
+        name: str,
         display_name: Optional[str] = None,
```

Comment:
Here too - the `Document` dataclass does not allow optional `display_name`, is it OK that this does? (e.g. if the API always returns a string then this would be OK - I don't know how it works so am just checking)

Reply:
If you just set the `name` for a `Document` or `Corpus`, the client side API should automatically just generate a `display_name`. When we unpack the object, the `display_name` is included then.